letsencrypt/client/CONFIG.py

```diff
@@ -116,3 +116,6 @@
 
 APACHE2 = "/etc/init.d/apache2"
 """Command used for reload and restart."""
+
+# TODO: add dnsutils to required/suggested packages
+NSUPDATE_CMD = "nsupdate -k le-nsupdate.key"  # see "man nsupdate"
```

Comment:
@ThomasWaldmann this is the offending line.

Note, that although `master` does not yet have #220 merged, `CONFIG` was always meant to be editable by users. Even if not (we might argue how hard it is to edit Python code), then this creates unnecessary vulnerability I will have to take care of when merging #220, and since it's your code, fixing it now would be awesome (you get the attribution as well!). Thanks :)